### PR TITLE
Chase referee 

### PR DESCRIPTION
**Pull request by adamsilver**
image
image
image

### CODE REVIEW

File: app/views/applications/chase-referee.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{% endblock %}
+
+{% block content %}
+
+  <form action="/applications/chase-referee-success">
+
+    <h1 class="govuk-heading-xl">Are you sure you want to send an email to chase the referee?</h1>
+
+    {{ govukButton({
+      text: "Yes I’m sure - send the email"
```
Comment (fofr) [inline app/views/applications/chase-referee.html]:
```suggestion
      text: "Yes - send the email"
```

File: app/views/applications/chase-referee-success.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Success — Chase referee - {{serviceName}}
+{% endblock %}
+
+{% block content %}
+
+  {{ govukPanel({
+    titleText: "Email sent to referee"
```
Comment (fofr) [inline app/views/applications/chase-referee-success.html]: Wonder if this should be a success banner back on the original page.

Comment (fofr) [reply app/views/applications/chase-referee-success.html]: Panels are more for end of service/journey flows. This is more end of a specific task.

Comment (adamsilver) [reply app/views/applications/chase-referee-success.html]: I wonder the same.

Comment (chubberlisk) [reply app/views/applications/chase-referee-success.html]: Theo: "Yes, a flash message seems more appropriate"

Comment (adamsilver) [reply app/views/applications/chase-referee-success.html]: Have changed to a flash.

File: app/views/applications/chase-referee.html (hunk 1 above)
Comment (adamsilver) [reply app/views/applications/chase-referee.html]: Done

**Pull request closed without merging**